cellxgene_schema_cli/cellxgene_schema/utils.py

```diff
@@ -2,17 +2,21 @@
 import os
 import sys
 from base64 import b85encode
+from functools import lru_cache
 from typing import Dict, List, Union
 
 import anndata as ad
 import numpy as np
+from cellxgene_ontology_guide.ontology_parser import OntologyParser
 from scipy import sparse
 from xxhash import xxh3_64_intdigest
 
 logger = logging.getLogger(__name__)
 
 SPARSE_MATRIX_TYPES = {"csc", "csr", "coo"}
 
+ONTOLOGY_PARSER = OntologyParser(schema_version="v5.3.0")
```

Comment:
i would prefer passing in the `ONTOLOGY_PARSER` that we create an instance of in `validate.py` into `is_ontological_descendant_of`, that way we only have one place to update when we bump the schema version

Reply:
yes, I see your point. I definitely agree we should minimize (to 1) the number of places a bump needs to be reflected. I'll do that.

One question I have is, why is the version in the code at all and not as a configuration option?

Reply:
this was previously set to something like `get_current_schema_version()`. however, the current schema version is 5.2.0, because we bump the version number as part of the automated release process. and in the past, we would usually bump the ontology versions in COG completing all the feature changes to the validator. there's some more context in [this slack thread](https://czi-sci.slack.com/archives/C07AV4NU9D2/p1732047031943819) but basically i decided hard-coding this was easier than needing to change some of our automated release process to make sure we're not double-bumping the version number before creating the new 5.3.0 release.

Reply:
I've followed your suggestion. Good looking out @joyceyan